src/proposals/mips/mip-b08/MTokens.json

```diff
@@ -0,0 +1,21 @@
+[
+    {
+        "addressesString": "MOONWELL_wstETH",
+        "supplyCap": 20000e18,
```

Comment:
Awaiting Warden and Gauntlet's final recommendation, but an initial draft recommendation from Warden is as follows:
> Considering wstETH will be heavily arbitraged off-chain and between chains, we’d be comfortable to start with 1000 supply cap ($2.12M) and 500 borrow cap ($1.06M)